src/Interfaces/IOpdexMinedToken.cs

```diff
@@ -53,6 +56,12 @@ public interface IOpdexMinedToken : IStandardToken256
     /// </summary>
     string Name { get; }
 
+    /// <summary>
+    /// Retrieves the next block that distribution is available.
+    /// </summary>
+    /// <returns>The block height of the next distribution.</returns>
+    ulong NextDistributionBlock();
```

Comment:
nitpick: This could be a property

Reply:
resolved in https://github.com/Opdex/opdex-governance/pull/4/commits/6b111e1317d1317d17c77145a7f337592a12e39d